src/components/Table/TableCell/TableCell.module.scss

```diff
@@ -0,0 +1,9 @@
+.tableCell {
+  padding: 9px 16px;
+  font: var(--font-text-small-normal);
+  line-height: var(--font-line-height-20);
+  color: var(--primary-text-color);
+  white-space: nowrap;
+  text-overflow: ellipsis;
+  overflow: hidden;
```

Comment:
I see that in `TableCell` you're already making the condition by using the `Text` component, so I think this code can be deleted